.github/workflows/release.yml

```diff
@@ -3,28 +3,48 @@ name: Release
 on:
   push:
     tags:
-      - 'v*'
+      - 'v[0-9]*'
```

Comment:
Couldnt find out on the Github Docs if this is proper regex? If it is, this wouldnt match semver notation. It's probably just a wildcard for anything I assume?

Reply:
Good question! It's not proper regex, see https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet 👍

Reply:
Also I've tested it in a separate repo and it worked with SemVer ✅